From 0d519a440a0343a6d8690e89280ef6df826411c6 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 5 Aug 2022 07:56:59 -0600
Subject: [PATCH] Bump Security String to 2022-08-05

Implemented:
============
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2022-20344  A-232541124    EoP    High       10, 11, 12, 12L
CVE-2022-20345  A-230494481    RCE    Critical   12, 12L
CVE-2022-20346  A-230493653    ID     High       10, 11, 12, 12L
CVE-2022-20347  A-228450811    EoP    High       10, 11, 12, 12L
CVE-2022-20348  A-228315529    EoP    High       10, 11, 12, 12L
CVE-2022-20349  A-228315522    EoP    High       10, 11, 12, 12L
CVE-2022-20350  A-228178437    ID     High       10, 11, 12, 12L
CVE-2022-20352  A-222473855    ID     High       12, 12L
CVE-2022-20353  A-221041256    ID     High       10, 11, 12, 12L
CVE-2022-20354  A-219546241    EoP    High       11, 12, 12L
CVE-2022-20355  A-219498290    DoS    High       10, 11, 12, 12L
CVE-2022-20356  A-215003903    EoP    High       11, 12, 12L
CVE-2022-20357  A-214999987    ID     High       12, 12L
CVE-2022-20358  A-203229608    ID     High       10, 11, 12, 12L
CVE-2022-20360  A-228314987    EoP    High       10, 11, 12, 12L
CVE-2022-20361  A-231161832    EoP    High       10, 11, 12, 12L

Not Implemented:
=======================
None

Not Applicable (platform source):
=================================
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2021-39696  A-185810717    EoP    High       10, 11, 12

Change-Id: Icd3fe124adbcdb9e3d4bfe9f550e11df6bee31cf
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 51a14bbc49..eeb4134b80 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -240,7 +240,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2022-07-05
+    PLATFORM_SECURITY_PATCH := 2022-08-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.37.1

